src/ralph/data_importer/resources.py

```diff
@@ -17,8 +17,7 @@
 )
 from ralph.data_importer.widgets import (
     AssetServiceEnvWidget,
-    BaseObjectLicenceWidget,
-    BaseObjectManyToManyWidget,
+    BaseObjectThroughWidget,
     BaseObjectWidget,
     ImportedForeignKeyWidget,
     LicenceUserWidget,
```

Comment:
btw whats the difference between `LicenceUserWidget` and `UserManyToManyWidget`?